### ISSUE TITLE
Currently not installable via composer

### ISSUE THREAD

**Issue by utrotzek** [Tuurlijk/typo3scan#31]
When tryining to install the package with

```bash
composer req --dev michielroos/typo3scan
```
it ends up with the following error:

```
michielroos/typo3scan 1.6.3 requires typo3/cms-scanner dev-master -> no matching package found.
```

The package typo3/cms-scanner seems to be removed, because I cannot find it on packagist.org.

**Comment by LeoVie** [comment]
I guess, this was never on packagist, see https://github.com/Tuurlijk/typo3scan#installation
You have to add this github repository to your composer configuration.

**Comment by Tuurlijk** [comment]
You can add a repository for it in your composer.json:

`    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/Tuurlijk/scanner"
        }
    ],`

**Comment by Cabag** [comment]
Hi,
That is not the solution since the package typo3/cms-scanner which is maintained by @ohader, cannot be found. I tried adding Olivers Repo as well, but no success. My repository section in the composer.json looks like this:
```
"repositories": [
	{"type": "vcs", "url": "https://github.com/ohader/scanner"},
	{"type": "vcs", "url": "https://github.com/Tuurlijk/scanner"}
],
```
But I am still getting the error 
`michielroos/typo3scan 1.6.6 requires typo3/cms-scanner dev-master -> no matching package found.`

I think the real problem is with olivers package, but since I stumbled upon this, I thought, I'd leave a comment.

**Comment by Tuurlijk** [comment]
@mbrodala suggested to split the package into a binary distributed through packagist.

Next to that there will be a source package with the content of this repo.

That would make it installable and standalone.

**Comment by Tuurlijk** [comment]
Fixed with release 1.7.2

**Comment by mbrodala** [comment]
Thanks a lot, works like a charm!

```
$ composer req --dev michielroos/typo3scan
$ vendor/bin/typo3scan
   ________  ______  ____ _____
  /_  __/\ \/ / __ \/ __ \__  /   ______________ _____
   / /    \  / /_/ / / / //_ <   / ___/ ___/ __ `/ __ \
  / /     / / ____/ /_/ /__/ /  (__  ) /__/ /_/ / / / /
 /_/     /_/_/    \____/____/  /____/\___/\__,_/_/ /_/

        https://github.com/tuurlijk/typo3scan

          Hand coded with ♥️ by Michiel Roos 
...
```

:+1:
